initPTB.m

```diff
@@ -64,36 +48,13 @@
 % Get the screen numbers and draw to the external screen if avaliable
 cfg.screen = max(Screen('Screens'));
 
-% init PTB with different options in concordance to the Debug Parameters
-if cfg.debug
-
-    % set to one because we don not care about time
-    Screen('Preference', 'SkipSyncTests', 2);
-    Screen('Preference', 'Verbosity', 0);
-    Screen('Preferences', 'SuppressAllWarnings', 2);
-
-    if cfg.testingSmallScreen
-        [cfg.win, cfg.winRect] = Screen('OpenWindow', cfg.screen, cfg.backgroundColor,  [0,0, 480, 270]);
-    else
-        if cfg.testingTranspScreen
-        PsychDebugWindowConfiguration
-        end
-        [cfg.win, cfg.winRect] = Screen('OpenWindow', cfg.screen, cfg.backgroundColor);
-    end
-
-else
-    Screen('Preference','SkipSyncTests', 0);
-    [cfg.win, cfg.winRect] = Screen('OpenWindow', cfg.screen, cfg.backgroundColor);
-
-end
-
+cfg = openWindow(cfg);
 
 % window size info
 [cfg.winWidth, cfg.winHeight] = WindowSize(cfg.win);
 
 
 
-
 % ---------- FIX LATER ---------- %
```

Comment:
should we add this as an issue so that we don't forget?

Reply:
done